### DISCUSSION TITLE
How to download multiple YouTube videos with URLs List?

### DISCUSSION THREAD

**Discussion by Flexes35** [deniscerri/ytdlnis#519] [Q&A] [answered]
I have a list of URLs for 142 YouTube videos and want to batch download them all at once.

**Comment by teddysulaimanGL** [comment, marked as answer]
"[you can create a txt file and fill it with links/playlists/search queries separate by a new line and the app will process them](https://github.com/deniscerri/ytdlnis#:~:text=you%20can%20create%20a%20txt%20file%20and%20fill%20it%20with%20links/playlists/search%20queries%20separate%20by%20a%20new%20line%20and%20the%20app%20will%20process%20them)"

Pass that file using the share menu, just like how you usually share a single URL

**Reply by grf11-git** [reply]
Hi, how do you set the default in YTDLnis to force the output to be saved in video format ( rather than audio!).
Looks like I need a command template to allow me to set the defaults.
Not sure what I need, but something like ... 
Output folder ...
 (artist)/(uploader) 
Filename ...
%(playlist_index,playlist_autonumber&{}. |)s - %(title)s
Extention... 720p mp4
(Id 398)

Any chance someone could provide the parameters I would need to add to the ftemplate command field in the android app?

**Reply by deniscerri** [reply]
Just go to folder settings and set filename template for video

**Comment by hussenmuhdin49-byte** [comment]
[[YTDLnis Log] Sheikh Khalid al Rashid amharic.txt](https://github.com/user-attachments/files/22243917/YTDLnis.Log.Sheikh.Khalid.al.Rashid.amharic.txt)

**Comment by hussenmuhdin49-byte** [comment]
Support

**Reply by deniscerri** [reply]
You have set preferred subtitle format to srt but the video is in webm format, webm doesnt work with srt subtitles. Maybe leave subtitle format to default
